{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -268,3 +269,10 @@
 EMAIL_USE_TLS = email_config["EMAIL_USE_TLS"]
 EMAIL_USE_SSL = email_config["EMAIL_USE_SSL"]
 DEFAULT_FROM_EMAIL = get_env_variable("EMAIL_FROM", "webmaster@localhost")
+
+
+#################
+# Changelog gen #
+#################
+
+CHANGELOG_MD_PATH = os.path.join(BASE_DIR, "CHANGELOG.md")
```

Comment:
Should we also expose a changelog view in the project template ? Or do you think that this is a project-specific decision ?